ui/pages/confirmations/components/gas-details-item/gas-details-item.js

```diff
@@ -148,7 +145,7 @@ const GasDetailsItem = ({
                 }}
                 type={SECONDARY}
                 value={getTransactionFeeTotal}
-                hideLabel={Boolean(useNativeCurrencyAsPrimaryCurrency)}
+                hideLabel // Sending hideLabel true by default because this should no longer depend on useNativeCurrencyAsPrimaryCurrency and will always display fiat value
```

Comment:
```suggestion
                hideLabel
                // Label not required here as it will always display fiat value.
```

Reply:
As someone without context about `useNativeCurrencyAsPrimaryCurrency`, this could be confusing